runtime/lua/vim/health.lua

```diff
@@ -106,6 +106,7 @@
 local M = {}
 
 local s_output = {} ---@type string[]
+local report_output = {} ---@type { ['ok']: number, ['warn']: number, ['error']: number }
```

Comment:
Not sure of the best name for this, but `report_output` seems strange.
```suggestion
local results = {} ---@type { ['ok']: number, ['warn']: number, ['error']: number }
```

Reply:
Isn't 'results' a bit vague? Maybe check_summary, msg_counts, check_counts?

Reply:
Changed to `check_summary`.